config/jobs/kubernetes/sig-cloud-provider/azure/sig-azure-config.yaml

```diff
@@ -294,12 +294,14 @@ periodics:
     - image: gcr.io/k8s-testimages/kubekins-e2e:v20200513-7c0e31a-master
       command:
       - "runner.sh"
-      - "./scripts/ci-conformance.sh"
+      - "./scripts/ci-entrypoint.sh"
```

Comment:
are there other occurrences of ci-conformance.sh left in sig-cluster-lifecycle jobs? If so, will you change those in a separate PR or should they be changed in this one too?

Reply:
Good catch, I noticed that `cluster-api-provider-azure-periodics.yaml` is still using `ci-conformance.sh`. I will update this PR. Other than that, I don't think there are any occurrences of `ci-conformance.sh` left in CAPZ-related configs.